addons/mrp/models/stock_move.py

```diff
@@ -111,6 +111,8 @@ class StockMove(models.Model):
     cost_share = fields.Float(
         "Cost Share (%)", digits=(5, 2),  # decimal = 2 is important for rounding calculations!!
         help="The percentage of the final production cost for this by-product. The total of all by-products' cost share must be smaller or equal to 100.")
+    qty_available = fields.Float(related='product_id.qty_available')
```

Comment:
maybe use product_ before variable name to avoid the warning on runbot and the source of the field is clear

Reply:
fields prefixed with product_,
labels from product added (prefixed with Product)
labels added on fields in view